dipy/reconst/csdeconv.py

```diff
@@ -118,18 +120,20 @@ def __init__(self, gtab, response, reg_sphere=None, sh_order=8, lambda_=1,
         # scale lambda_ to account for differences in the number of
         # SH coefficients and number of mapped directions
         # This is exactly what is done in [4]_
-        self.lambda_ = (lambda_  * self.R.shape[0] * r_rh[0] /
-                        (np.sqrt(self.B_reg.shape[0]) * np.sqrt(362.))
-                       )
+
+        lambda_ = (lambda_  * self.R.shape[0] * r_rh[0] /
+                   (np.sqrt(self.B_reg.shape[0]) * np.sqrt(362.)))
```

Comment:
Or why not just fix it instead, if the old historical version is a hardcoded value just because (but is it related to the number of points of the sphere, that's what I was curious about).